plugins/Jobs.cpp

```diff
@@ -98,11 +100,10 @@ void BedrockPlugin_Jobs::upgradeDatabase(SQLite& db) {
                                "retryAfter  TEXT NOT NULL DEFAULT \"\")",
                            ignore));
 
-    // These indexes are not used by the Bedrock::Jobs plugin, but provided for easy analysis
-    // using the Bedrock::DB plugin.
-    SASSERT(db.write("CREATE INDEX IF NOT EXISTS jobsName     ON jobs ( name     );"));
-    SASSERT(db.write("CREATE INDEX IF NOT EXISTS jobsParentJobIDState ON jobs ( parentJobID, state ) WHERE parentJobID != 0;"));
-    SASSERT(db.write("CREATE INDEX IF NOT EXISTS jobsStatePriorityNextRunName ON jobs ( state, priority, nextRun, name );"));
+    // These indexes are not used by the Bedrock::Jobs plugin, but provided for easy analysis using the Bedrock::DB plugin.
```

Comment:
Porting from another comment: I still don't get what this is saying, @coleaeason @tylerkaraszewski ? I think we can kill this comment?

Reply:
yeah we can kill it, i am nearly certain these are used.

Reply:
Removed.